### DISCUSSION TITLE
Hacktoberfest Contributions? 🎃

### DISCUSSION THREAD

**Discussion by MarkBennett** [keystonejs/keystone#6736] [Questions] [answered]
Don't want to step on any toes or start any battles here, but also keen to get a Hacktoberfest shirt again!

I was just curious if Keystone was accepting PRs as part of Hacktoberfest this year? If so I believe they're asking maintainers to opt-in this year:

https://hacktoberfest.digitalocean.com/resources/maintainers

Thanks so much and really appreciate the work you do!

**Comment by timleslie** [comment, marked as answer]
Hi @MarkBennett, thanks for bringing this to our attention. We're in the middle of preparing for a major release of Keystone, so we don't have the capacity right now to focus on other initiatives. As such we won't be participating in Hacktoberfest this year. Of course, contributions of bug fixes, bug reports, or feature requests are always open, and the [community slack](https://community.keystonejs.com/) has loads of people sharing their experiences 👍

**Reply by MarkBennett** [reply]
Sounds good! Thanks for getting back to me!
